### ISSUE TITLE
Automated label changes should match acting user

### ISSUE THREAD

**Issue by ezra-g** [codetree/feedback#168]
I closed an issue in Github, so Codetree removed our `status: in sprint` label, which is super handy. However, it removed the label using @awaxman11's Github account rather than my own. It seems ideal for Codetree to perform the automated action using the account of the user who triggered label change to avoid confusion. Since starting to use Codetree, we've had a few cases where people were confused about why @awaxman11 was updating issues when really it was Codetree doing awesome stuff on someone else's behalf :).

![_admin__implement_the_ui_for_the_users_show_page__8__ _issue__12002_ _seatgeek_tixcast](https://cloud.githubusercontent.com/assets/349595/11002713/81ef2644-847c-11e5-8f42-846ae7565321.jpg)

**Comment by derrickreimer** [comment]
Yeah, we should probably be more intelligent about that in cases where it's feasible.

**Comment by wanderingmatt** [comment]
:+1:
